File: app/src/main/java/br/com/pauloc/mercadin/DB/ProdutoContract.java
package br.com.pauloc.mercadin.DB;

import android.provider.BaseColumns;

public class ProdutoContract {

    private String TABELA_PRODUTO = "produto";

    static final class ProdutoColumns implements BaseColumns{
        static String DESCRICAO = "descricao";
        static String QNT_ITEM = "qnt_item";
        static String MARCA = "marca";
        static String CATEGORIA = "categoria";
        static String VALOR = "valor";
        static String VALIDADE = "Validade";
    }
}

File: app/src/main/java/br/com/pauloc/mercadin/DB/ProdutoContract.kt
package br.com.pauloc.mercadin.DB

import android.provider.BaseColumns

class ProdutoContract {
    private val TABELA_PRODUTO = "produto"

    internal object ProdutoColumns : BaseColumns {
        var DESCRICAO = "descricao"
        var QNT_ITEM = "qnt_item"
        var MARCA = "marca"
        var CATEGORIA = "categoria"
        var VALOR = "valor"
        var VALIDADE = "Validade"
    }
}